doc/Tutorials/index.rst

```diff
@@ -9,7 +9,7 @@ you may also try out live copies on `mybinder.org
 explore many other examples in our `contrib repository
 <https://github.com/ioam/holoviews-contrib>`_. Lastly, for the most
 responsive experience, you can install HoloViews and try out the
-`tutorial notebooks`_ for yourself.
+tutorial notebooks for yourself.
 
```

Comment:
This isn't a broken link - it gets replaced with the right link based on the current version in setup.py. See the replacement [here](https://github.com/ioam/holoviews/blob/master/doc/conf.py#L31).